app/src/pages/[resourceType]/[id].tsx

```diff
@@ -80,6 +98,50 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     }
   );
 
+  //Sorts dependencies alphabetically based on their display property. If the the
+  //dependency is a library/measure, however, it will always take precedence in sorting to ensure all
+  //dependencies with links are always shown first
+  const sortDependencies = useCallback(() => {
```

Comment:
Due to the useEffect on line 118-129, the search results will display all dependencies whenever the sortDependencies variable is called. So if the user searches something then it will display the results briefly and then revert back to showing all dependencies at once. The way I remedied this is by using the useCallBack to help make sure the search functionality actually works.